api/capi/src/nntrainer_util.cpp

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (C) 2020 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
Better to use SPDX for the new file.

Reply:
I don't understand about SPDX - how are we using this. As per https://lwn.net/Articles/739183/, it can be added to any file. Can you elaborate on whats nntrainer policy for SPDX to be used?

Reply:
``` 
/**
 * Copyright (C) 2020 Parichay Kapoor <pk.kapoor@samsung.com>
 *
 * SPDX-License-Identifier: Apache-2.0-only
 *
 * @file nntrainer_util.cpp
 * @date 10 July 2020
 * @brief NNTrainer/Utilizer C-API Wrapper.
 * @see	https://github.com/nnstreamer/nntrainer
 * @author Parichay Kapoor <pk.kapoor@samsung.com>
 * @bug No known bugs except for NYI items
 */
```

Reply:
https://spdx.dev/specifications/

No..

The appropriate formating will be:
```
// SPDX-License-Identifier: Apache-2.0-only
/**
 * Copyright (C) 2020 Parichay Kapoor <pk.kapoor@samsung.com>
 *
 * @file nntrainer_util.cpp
 * @date 10 July 2020
 * @brief NNTrainer/Utilizer C-API Wrapper.
 * @see	https://github.com/nnstreamer/nntrainer
 * @author Parichay Kapoor <pk.kapoor@samsung.com>
 * @bug No known bugs except for NYI items
 */
```

Reply:
You are supposed to add SPDX identifier at the top (or nearest position to the top).

Reply:
Done. Is the plan to change the license across nntrainer to SPDX?

Reply:
Not all, for the newest one.

Reply:
Got it.